dist/autocomplete.jquery.js

```diff
@@ -372,11 +372,12 @@
 	  $input = this.$node.find(_.className(this.cssClasses.prefix, this.cssClasses.input));
 	  $hint = this.$node.find(_.className(this.cssClasses.prefix, this.cssClasses.hint));
 
-	  if (o.dropdownMenuContainer) {
-	    DOM.element(o.dropdownMenuContainer)
-	      .css('position', 'relative') // ensure the container has a relative position
-	      .append($menu.css('top', '0')); // override the top: 100%
-	  }
+	  DOM.element('body').append($menu);
+	  window.$input = $input;
```

Comment:
No no, that's debug purpose. Don't spend too much time here; we're gonna iterate on that a lot.